### DISCUSSION TITLE
Webhook V2: Slack [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5922] [Discord Threads]
> 🙋‍♂️ **romek_rozen**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### V2- customising message is not possible. Default message is sent for corresponding trigger using webhook URL created from Slack  
#  
> 🙋‍♂️ **romek_rozen**  
#### OK 🙂 thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1106171386281599046)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
